GitHub DS Plugin docs (#292)
grafana/github-datasource: docs/sources/variables-and-macros/_index.md

Reviewer: Does this section provide extra information specific to the plugin over the Grafana documentation on the same topics like https://grafana.com/docs/grafana/latest/dashboards/variables/?

I actually can't find macros documentation in Grafana so perhaps that's specific to plugins?

Reviewer: Yeah, I'd definitely keep the Macros content since it's only in plugins and this information is specific to this plugin.

Author: fixed now

Reviewer: I think this should use the concept template described in https://grafana.com/docs/writers-toolkit/structure/topic-types/concept/

Author: done

---

> title: 'Variables & macros'
> menuTitle: Variables & macros
> description: Variables & macros
> keywords:
>   - data source
>   - github
>   - github repository
>   - API
> labels:
>   products:
>     - oss
>     - enterprise
>     - cloud
> weight: 200
> ---
> 
> # Variables & macros

Reviewer: ```suggestion
title: Variables and macros
menuTitle: Variables and macros
description: Learn about variables and macros you can use in the GitHub data source plugin for Grafana
keywords:
  - data source
  - github
  - github repository
  - API
labels:
  products:
    - oss
    - enterprise
    - cloud
weight: 200
---

# Variables and macros
```

> - [Variables](./variables)
> - [Macros](./macros)

Reviewer: ```suggestion
To learn more about variables and macros you can use in the GitHub data source plugin for Grafana, refer to the following topics:

{{< section menuTitle="true" withDescriptions="true" >}}
```

---

> title: 'Variables & Macros'
> menuTitle: Variables & Macros
> description: Variables & Macros

Reviewer 2: ```suggestion
title: 'Variables & macros'
menuTitle: Variables & macros
description: Variables & macros
```

aliases:
  - github
keywords:
  - data source
  - github
  - github repository
  - API
labels:
  products:
    - oss
    - enterprise
    - cloud
weight: 200
---

> # Variables & Macros

Reviewer 2: ```suggestion
# Variables & macros
```
